ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -130,14 +137,37 @@ private T GetSetting<T>(string key, T defaultValue)
                     return (T)data;
                 }
             }
+#else
+            if (typeof(T) == typeof(bool))
+            {
+                Boolean result = Boolean.TryParse(ConfigurationManager.AppSettings[key], out result);
```

Comment:
We probably want to change this line to:
`return (T)Boolean.Parse(ConfigurationManager.AppSettings[key]);`